lib/online_order.rb

```diff
@@ -0,0 +1,104 @@
+require 'csv'
+require 'awesome_print'
+require 'money'
+require_relative '../lib/order'
+require_relative '../lib/customer'
+I18n.enforce_available_locales = false
+
+module Grocery
+  class OnlineOrder < Order
+    attr_reader :id, :products, :customer, :status
```

Comment:
Since `OnlineOrder` is a subclass of `Order` you don't need to define an `id`, or `products` helper.  Instead just use the ones you inherit from `Order`.

This should be:
```ruby
    attr_reader :customer, :status
```